pkg/cpio/utils.go

```diff
@@ -34,7 +33,7 @@ func StaticRecord(contents []byte, info Info) Record {
 func StaticFile(name string, content string, perm uint64) Record {
 	return StaticRecord([]byte(content), Info{
 		Name: name,
-		Mode: unix.S_IFREG | perm,
+		Mode: S_IFREG | perm,
```

Comment:
I don't see these constants defined anywhere, did you forget to commit a file?

Reply:
oops. I did.